tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator/ExtensionsMetadataGenerator.csproj

```diff
@@ -12,14 +12,18 @@
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
     <Description>Azure Functions extensions metadata generator</Description>
     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)..\..\..\..\src.ruleset</CodeAnalysisRuleSet>
+    <IsPackable>true</IsPackable>
+    <BeforePack>Build;PackReferenceAssemblies</BeforePack>
+    <CommitHash Condition="$(CommitHash) == ''">N/A</CommitHash>
+    <InformationalVersion>Commit hash: $(CommitHash)</InformationalVersion>
```

Comment:
A lot of these properties are being copied from other projects. Are these good candidates for the common props file that was introduced?

Reply:
It's really just these 2 -- and they're actually slightly different b/c we don't have some of the same properties as the other projects (https://github.com/Azure/azure-functions-host/blob/dev/build/common.props#L12-L13). I was going to share all of that in the package.props, but it'd end up making this version start with "2.0", which seemed weird -- it started getting complicated just to save 4 lines of markup, so I just did this :-)